backend/src/bundles/hard-skills/hard-skills.model.ts

```diff
@@ -5,27 +5,22 @@ import {
     DatabaseTableName,
     HardSkillsTableColumn,
     TalentHardSkillsTableColumn,
-    UserDetailsTableColumn,
 } from '~/common/packages/database/database.js';
 
 class HardSkillsModel extends AbstractModel {
     public 'name': string;
 
     public static override get tableName(): string {
-        return DatabaseTableName.TALENT_HARD_SKILLS;
+        return DatabaseTableName.HARD_SKILLS;
     }
 
     public static override relationMappings = {
-        talents: {
-            relation: Model.ManyToManyRelation,
+        talentSkills: {
+            relation: Model.HasManyRelation,
             modelClass: '',
             join: {
                 from: `${DatabaseTableName.HARD_SKILLS}.${HardSkillsTableColumn.ID}`,
-                through: {
-                    from: `${DatabaseTableName.TALENT_HARD_SKILLS}.${TalentHardSkillsTableColumn.HARD_SKILL_ID}`,
-                    to: `${DatabaseTableName.TALENT_HARD_SKILLS}.${TalentHardSkillsTableColumn.USER_DETAILS_ID}`,
-                },
-                to: `${DatabaseTableName.USER_DETAILS}.${UserDetailsTableColumn.ID}`,
+                to: `${DatabaseTableName.TALENT_HARD_SKILLS}.${TalentHardSkillsTableColumn.HARD_SKILL_ID}`,
             },
         },
```

Comment:
I think you don't need this relation

Reply:
I added it because I saw the relation in migration.
Done.